accounts/static/accounts/css/onboarding_movies.css

```diff
@@ -111,4 +111,35 @@
     background-color: #9a8aff;
     transform: scale(1) !important;
   }
+
+}
+
+@media screen and (max-width: 500px) {
+  #help_modal{
+    width:100% !important; /*TODO: overrides separate !important width in modal.css  */
+    padding: 20px 0px;
+  }
+}
+
+#help_button {
+  position: fixed;
+  right: 20px;
+  bottom: 100px;
+  background-color: #816efa;
+  border-radius: 100%;
+  padding: 1.25rem;
+  color:white;
+  font-size: 1.5rem;
+}
```

Comment:
no spacing! :0

i am looking for a css linter that we can use, cause apparently prettier doesn't handle this